src/runtime/string.cpp

```diff
@@ -210,6 +210,8 @@ static PRIMFN(prim_read) {
       runtime.heap.reserve(need_pass);
 
       String *out = String::claim(runtime.heap, size);
+      out->c_str()[size] = 0;
+
```

Comment:
nit: You don't need this anymore

Reply:
I want to keep it for code consistency until we make `c_str()` const only. Every other case in this file still has this